### PR TITLE
Fix non-functional linting warnings

### PR DESCRIPTION
**Pull request by gatici**
Fixing whitespace and import order warnings by running following command. This PR does not affect the code functionality.
```
golangci-lint run --fix
```

### CODE REVIEW

**Comment by gab-arrobo** [comment]
retest this please

**Comment by thakurajayL** [comment]
Do you need this to be reviewed ?

**Comment by gab-arrobo** [comment]
test this please

**Comment by gab-arrobo** [comment]
retest this please

**Comment by gab-arrobo** [comment]
@gatici, I know why the `pull_request` fails and you have a PR that address the `unit-test-pfcpiface` (#180), but I was expecting this PR to pass the `golangci-lint` test/GHA. Am I overlooking something?

File: context/sm_context.go [hunk 1]
```diff
@@ -9,25 +9,18 @@ package context
...
-	"github.com/omec-project/smf/metrics"
-	"github.com/omec-project/smf/msgtypes/svcmsgtypes"
-	"github.com/omec-project/smf/qos"
-	errors "github.com/omec-project/smf/smferrors"
-	"github.com/omec-project/smf/transaction"
-	"github.com/sirupsen/logrus"
-
 	"net"
 	"net/http"
+	"net"
```
Comment (gab-arrobo) [inline context/sm_context.go:14]:
```suggestion
```

File: context/sm_context.go [hunk 2]
```diff
@@ -9,25 +9,18 @@ package context
...
+	"net/http"
```
Comment (gab-arrobo) [inline context/sm_context.go:15]:
```suggestion
```

**Comment by gab-arrobo** [comment]
> @gatici, I know why the `pull_request` fails and you have a PR that addresses the `unit-test-pfcpiface` (#180), but I was expecting this PR to pass the `golangci-lint` test/GHA. Am I overlooking something?

@gatici, please double check this PR because it like `golangci-lint run --fix` did not properly fix linting warnings and actually added more issues. For example, in context/sm_contest.go, imports are now duplicated as shown below:
```
	"net"
	"net/http"
	"net"
	"net/http"
```
I am making a couple modifications to see if the issue can be directly addressed from the PR

**Comment by gab-arrobo** [comment]
test this please

File: consumer/nnrf.go [hunk 3]
```diff
@@ -187,8 +186,8 @@ func getSvcMsgType(nfType models.NfType) svcmsgtypes.SmfMsgType {
 }
 
 func SendNrfForNfInstance(nrfUri string, targetNfType, requestNfType models.NfType,
-	param *Nnrf_NFDiscovery.SearchNFInstancesParamOpts) (models.SearchResult, error) {
-
+	param *Nnrf_NFDiscovery.SearchNFInstancesParamOpts,
```
Comment (gab-arrobo) [inline consumer/nnrf.go:189]: Why is this comma `,` added in here?

**Review by gab-arrobo — CHANGES_REQUESTED** [review]

**Comment by gab-arrobo** [comment]
> Fixing whitespace and import order warnings by running following command. This PR does not affect the code functionality.
> 
> ```
> golangci-lint run --fix
> ```

@gatici, how did you install `golangci-lint`? I installed it through snap and from go but, through neither of these two options, I cannot replicate your PR by running `golangci-lint run --fix`

**Comment by gab-arrobo** [comment]
> > golangci-lint
> 
> Hi @gab-arrobo I installed it through the [link](https://golangci-lint.run/usage/install/#binaries) by getting the binary, then run the command `golangci-lint run --fix`.

Hi @gatici, thanks for the link. Based on what I am reading, `golangci-lint` is the same as this [golangci-lint-action](https://github.com/golangci/golangci-lint-action) GHA, which is the recommended option for CI/CD projects as indicated [here](https://golangci-lint.run/usage/install/#github-actions). So, I am running the GitHub Action but it does not make any fixes/changes to the files. Would it be that your PR includes lots of changes because the way you are running the command (`golangci-lint run --fix`) and this does not take into account the options provided in [.golangci.yml](https://github.com/omec-project/smf/blob/master/.golangci.yml).

Moreover, when downloading the binary from the link you provided, I still cannot replicate your PR. Here are the steps I followed:

```
$ curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.55.2
$ golangci-lint --version
golangci-lint has version 1.55.2 built with go1.21.3 from e3c2265f on 2023-11-03T12:59:25Z
$ cd smf/
$ golangci-lint run --fix
$ git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
```

**Comment by gatici** [comment]
Hello  @gab-arrobo 

After rebasing on master, I can not reproduce it. The reason is https://github.com/omec-project/smf/pull/182/files# is disabled all the linters in .golangci.yml. After merging PR 182, this PR become invalid. I will close it at the moment.

**Pull request closed without merging**